src/projetos/ProjetoService.ts

```
@@ -33,6 +33,38 @@ class ProjetoService extends BaseService {
     return this.get<Projeto[]>('/meusprojetos')
   }
 
+  async count(): Promise<number> {
+    return this.get<number>('/count')
+  }
+
+  async findAllPaginated(page: number, size: number): Promise<{ data: Projeto[]; total: number }> {
```

ESLint

[error] Replace `page:·number,·size:·number` with `⏎····page:·number,⏎····size:·number,⏎··`

(prettier/prettier)